lib/Remote/HttpCommandExecutor.php

```diff
@@ -18,6 +18,11 @@ class HttpCommandExecutor implements WebDriverCommandExecutor
         'Accept: application/json',
     ];
 
+    /** @internal */
+    const SESSION_STORAGE = '/session/:sessionId/session_storage';
+    /** @internal */
+    const LOCAL_STORAGE = '/session/:sessionId/session_storage';
```

Comment:
This URL looks wrong to me.

Also, not sure that we should use const here at all.

Reply:
This was my edit of the original PR, and I made the mistake :facepalm: .

Const made sense to me, as this part is repeated for many times here.

Reply:
heh fair enough - it just drew my eye because it is the first time we've used a constant as part of the URL in the command.